packages/frontend/src/ee/features/aiCopilot/components/Admin/AiAgentAdminThreadsTable.tsx

```diff
@@ -0,0 +1,633 @@
+import {
+    type AiAgentAdminSortField,
+    type AiAgentAdminThreadSummary,
+} from '@lightdash/common';
+import {
+    ActionIcon,
+    Box,
+    Divider,
+    Group,
+    Paper,
+    Text,
+    TextInput,
+    Tooltip,
+    useMantineTheme,
+} from '@mantine-8/core';
+import {
+    IconArrowDown,
+    IconArrowsSort,
+    IconArrowUp,
+    IconBox,
+    IconClick,
+    IconClock,
+    IconMessageCircleStar,
+    IconRadar,
+    IconRobotFace,
+    IconSearch,
+    IconTextCaption,
+    IconUser,
+    IconX,
+} from '@tabler/icons-react';
+import {
+    MantineReactTable,
+    useMantineReactTable,
+    type MRT_ColumnDef,
+    type MRT_SortingState,
+    type MRT_Virtualizer,
+} from 'mantine-react-table';
+import {
+    useCallback,
+    useEffect,
+    useMemo,
+    useRef,
+    useState,
+    type UIEvent,
+} from 'react';
+import { LightdashUserAvatar } from '../../../../../components/Avatar';
+import MantineIcon from '../../../../../components/common/MantineIcon';
+import { useIsTruncated } from '../../../../../hooks/useIsTruncated';
+import SlackSvg from '../../../../../svgs/slack.svg?react';
+import { useInfiniteAiAgentAdminThreads } from '../../hooks/useAiAgentAdmin';
+
+const AiAgentAdminThreadsTable = () => {
+    const theme = useMantineTheme();
+    const [sorting, setSorting] = useState<MRT_SortingState>([
+        { id: 'createdAt', desc: true },
+    ]);
+    const [search, setSearch] = useState<string | undefined>(undefined);
```

Comment:
Search works with the data that's loaded already right?

Reply:
Correct, quickly adding search by title param - will follow same behaviour as `MetricsTable.tsx` and `InfiniteResourceTable`- thank you for pointing this out.